spec.bs

```diff
@@ -253,8 +258,8 @@ To <dfn>check interest group permissions</dfn> given an [=origin=]
 1. If |ownerOrigin| is [=same origin=] to |frameOrigin|, then return true.
 1. Let |permissionsUrl| be the result of [=building an interest group permissions url=]
   with |ownerOrigin| and |frameOrigin|.
-1. Let |request| be the result of [=creating a request=] with |permissionsUrl|, [=create a request/accept=]
-  set to "`application/json`"|. TODO: [maybe](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_permissions_checker.cc;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed;l=123) require cors mode here?
+1. Let |request| be the result of [=creating a request=] with |permissionsUrl|,
+  "`application/json`", and null.

```

Comment:
Just so I'm sure, this is a willful deviation from our implementation (which uses CORS) right? It seems that at a minimum, this request uses `no-cors` and an opaque initiator origin, whereas our implementation uses CORS and some origin [with an HTTPs scheme](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/interest_group/interest_group_permissions_checker.cc;l=89;drc=adac219925ef5d9c9a954d189c2e4b8852a4bbed).

Do we have a plan to fix this? Maybe this is part of a larger set of fixups we can do regarding the spec's overall Fetch integration, but what do you think?